certbot/compat/misc.py

```diff
@@ -2,42 +2,31 @@
 This compat module handles various platform specific calls that do not fall into one
 particular category.
 """
-import ctypes
-import errno
+from __future__ import absolute_import
+
+import os  # pylint: disable=os-module-forbidden
 import select
 import stat
 import sys
 
-from certbot import errors
-from certbot.compat import os
+try:
+    from win32com.shell import shell as shellwin32  # pylint: disable=import-error
+except ImportError:  # pragma: no cover
+    shellwin32 = None  # type: ignore
```

Comment:
This is my usual (probably overly paranoid) nit about not wanting to disable `mypy` when possible.

Reply:
Fixed as usual with a global boolean.